wordpress-mobile/WordPress-Android
# Swap Fabric for Sentry

**To test:**

Switch to this branch.

```bash
bundle exec fastlane run configure_apply
./gradlew clean assembleVanillaRelease
```

Run the binary on-device. Cause a crash, and see it appear!
